account_invoice_fixed_discount/models/account_move.py

```diff
@@ -71,7 +79,11 @@ def _get_price_total_and_subtotal_model(
         move_type,
     ):
         if self.discount_fixed != 0:
-            discount = ((self.discount_fixed) / price_unit) * 100 or 0.00
+            if price_unit >0.00:
```

Comment:
You should use float tools functions such as `float_compare` or `float_is_zero` whenever comparing float values.

Reply:
thanks @grindtildeath 
@gaelTorrecillas could you please correct this.
thanks